tests/product/test_server_install.py

```diff
@@ -171,6 +171,25 @@ def assert_installed_with_regex_configs(self, master, slaves):
                                            '/etc/presto/config.properties',
                                            self.default_workers_config_regex_)
 
+    def test_install_with_java8_home(self, dummy=True):
```

Comment:
can you also add a test for if you move where Java 8 is installed but don't set JAVA8_HOME?